### PR TITLE
add All DB Dao

### PR DESCRIPTION
**Pull request by sapozhnykov**

### CODE REVIEW

File: src/main/java/com/sapozhnykov/dao/auth/impl/AuthClientDBDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    public boolean add(long userId, String phone, String password) {
+        try ( Connection connection = DriverManager.getConnection(DB_URL, LOGIN, PASSWORD);
+              PreparedStatement statement = connection.prepareStatement("INSERT INTO AUTHCLIENT(USERID, PHONE, PASSWORD) VALUES(?, ?, ?)"))
+        {
+            statement.setLong(1, userId);
+            statement.setString(2, phone);
+            statement.setString(3, password);
+            int rows = statement.executeUpdate();
+
+            statement.close();
```
Comment (aleksashyn) [inline src/main/java/com/sapozhnykov/dao/auth/impl/AuthClientDBDaoImpl.java]: WTF? You are already use try-with-resources

Comment (sapozhnykov) [reply src/main/java/com/sapozhnykov/dao/auth/impl/AuthClientDBDaoImpl.java]: Fixed. Removed "close" here and in similar Dao

Comment (aleksashyn) [reply src/main/java/com/sapozhnykov/dao/auth/impl/AuthClientDBDaoImpl.java]: I still can see it

Comment (sapozhnykov) [reply src/main/java/com/sapozhnykov/dao/auth/impl/AuthClientDBDaoImpl.java]: Fixed. I just missed one

**Review by aleksashyn — APPROVED** [review]

**Pull request merged**